satpy/dataset/metadata.py

```diff
@@ -27,20 +27,19 @@
 from satpy.writers.utils import flatten_dict
 
 
-def combine_metadata(*metadata_objects, average_times=True):
+def combine_metadata(*metadata_objects):
     """Combine the metadata of two or more Datasets.
 
     If the values corresponding to any keys are not equal or do not
     exist in all provided dictionaries then they are not included in
-    the returned dictionary.  By default any keys with the word 'time'
-    in them and consisting of datetime objects will be averaged. This
-    is to handle cases where data were observed at almost the same time
-    but not exactly.  In the interest of time, lazy arrays are compared by
-    object identity rather than by their contents.
+    the returned dictionary.  The 'start_time' values will be set to the
+    earliest value and 'end_time' to latest time.  All other keys containing
+    the word 'time' are averaged.  Before these adjustments, non-datetime
+    objects are filtered out.  In the interest of time, lazy arrays are compared
+    by object identity rather than by their contents.
```

Comment:
Should the `time_parameters` also get this min/max treatment:

https://satpy.readthedocs.io/en/stable/reading.html#time-metadata

Reply:
I need to look at this in more detail later when I can concentrate a bit. I have a vague feeling that currently these are passed as-is.

Reply:
Should this docstring be updated to reflect the `time_parameters`? Or are `X_start_time` and `X_end_time` not treated with min/max?

Reply:
Yes. Updated in https://github.com/pytroll/satpy/pull/2737/commits/91dbc8e336af1c1830b6502f7accf488f1c90fd8